### PR TITLE
Homepage products

### PR DESCRIPTION
**Pull request by OmarAlithawi**

### CODE REVIEW

**Review by MiraAr — APPROVED** [review]

**Review by emrerdem1 — CHANGES_REQUESTED** [review]
Can you please run Prettier to get rid of unnecessary line-breaks in .scss file, every selector has to have only one line-break after closing curly braces according to what Louis said to me before. And there is spaces before commas such as `(12 ,1fr)`, it should be `(12, 1fr)`. If you can fix this too, that'd be great.

Other parts look great. :)

**Comment by OmarAlithawi** [comment]
PTAL

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: 2.1
+orbs:
+  node: circleci/node@1.1.6
+jobs:
+  build-and-test:
+    executor:
+      name: node/default
+    steps:
```
Comment (louisrli) [inline .circleci/config.yml]: this should not be in this pr

File: package.json [hunk 2]
```diff
@@ -25,7 +25,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint .  --ext .js"
+    "lint": "eslint '**/*.{js,jsx}'",
```
Comment (louisrli) [inline package.json]: normally if you're using eslint and prettier you will be combining the integrations (google prettier eslint)

for short term, delete lint, try adding it in another PR

File: package.json [hunk 3]
```diff
@@ -25,7 +25,8 @@
...
+    "pretty": "prettier --write \"./**/*.{js,jsx}\""
```
Comment (louisrli) [inline package.json]: this should include css files

File: src/components/home/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Products from '../products/Products';
+
+export default function Home() {
+	return (
+		<div>
+			<Products />
```
Comment (louisrli) [inline src/components/home/Home.jsx]: indentation is wrong

File: src/components/layout/productCard/ProductCard.jsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+import heart from '../../../assets/svgs/heart.svg';
+import { Col } from 'react-bootstrap';
+import './style.scss';
+import heartBroken from '../../../assets/svgs/heart-broken.svg';
+
+export default function ProductCard(props) {
+	const [isLiked, setIsLiked] = React.useState(false);
+
+	const settings = {
+		className: 'silderImages',
```
Comment (louisrli) [inline src/components/layout/productCard/ProductCard.jsx]: indentation still wrong please fix and i'll review this PR, write PTAL when ready

File: src/components/layout/productCard/style.scss [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+  .slick-slide{
+      display: inline-block;
+     
+  }
+
+
+  
+
+ 
+
```
Comment (louisrli) [inline src/components/layout/productCard/style.scss]: way too many newlines

**Review by louisrli — CHANGES_REQUESTED** [review]
not sure why the indentations are so big but please submit circleci in separate PR and fix the indentation and i'll look at this code

**Comment by louisrli** [comment]
not sure why there's a branch called homepage. all PRs should be into dev branch. talk to abdulrahman please if you need help fixing this and please set up circleci asap

**Comment by louisrli** [comment]
also you have two PRs with the exact same name

**Pull request closed without merging**